apps/web-giddh/src/app/proforma-invoice/proforma-invoice.component.html

```diff
@@ -275,7 +275,7 @@
                                     <span class="balance-due">{{isProformaInvoice || isEstimateInvoice ? 'Total Amount' : 'Balance Due'}}</span>
                                     <span class="sp-rupees" *ngIf="isMulticurrencyAccount">{{invFormData.accountDetails.currencySymbol}}
                                         <strong>
-                                            <ng-container *ngIf="adjustPaymentBalanceDueData">
+                                            <ng-container *ngIf="adjustPaymentData.totalAdjustedAmount">
```

Comment:
Use adjustPaymentData?.totalAdjustedAmount

Reply:
This is already defined in ts default as 0